README.md

```diff
@@ -16,9 +19,11 @@ signalfx-java will support this.
 
 ## Installation
```

Comment:
"Installation" is actually not super accurate, it's more like "How to use in your project" or something like that.